Scripts/AzPowerShellScript/AzHelper.psm1

```diff
@@ -196,13 +195,13 @@ function GetStorageAccount
     [CmdletBinding()]
     Param
     (
-        [Parameter(Mandatory=$false)]
+        [Parameter(Mandatory=$true)]
         [String]
-        $name = $script:StorageAccountName,
```

Comment:
Can we use the same parameter name for storageaccountname, or change all commandlets to have a name parameter and let the cmdlet clarify what object it reffers to

Reply:
Fixed in next patch